src/Microsoft.PowerShell.ConsoleGuiTools/TypeGetter.cs

```diff
@@ -169,7 +169,8 @@ public DataTable CastObjectsToTableView(List<PSObject> psObjects)
 
             foreach (var dataColumn in dataTableColumns)
             {
-                _cmdlet.WriteVerbose(dataColumn.ToString());
+                // See https://github.com/PowerShell/GraphicalTools/issues/192
+                //_cmdlet.WriteVerbose(dataColumn.ToString());
```

Comment:
Is this plan to bring it back eventually? Otherwise we should just delete the whole foreach loop.

Reply:
I agree. See #192